app/Resources/views/macros/forms.html.twig

```diff
@@ -81,7 +81,7 @@
             <span class="text-muted">({{ msg("optional") }})</span>
         {% endif %}
     </label>
-    <input type="date" class="form-control" name="{{ type }}" placeholder="2016-11-01"{% if required %} required{% endif %} />
+    <input type="date" class="form-control" name="{{ type }}" placeholder="YYYY-MM-DD"{% if required %} required{% endif %} />
```

Comment:
If not we might use `2016-12-31` since with `2016-11-01`, 11 could be interpreted as the day and 01 as the month